expo/tests/gbe/test_reports.py

```diff
@@ -37,15 +42,23 @@
 )
 from tests.factories.ticketing_factories import (
     TransactionFactory,
-    )
+)
+from tests.contexts import (
+    ActTechInfoContext,
+    ClassContext,
+    VolunteerContext,
+)
 import ticketing.models as tix
 
-import tests.functions.gbe_functions as functions
+from tests.functions.gbe_functions import (
+    grant_privilege,
+    login_as,
+)
 
 
 def _create_scheduled_show_with_acts(conference=None, qty=6):
     if not conference:
-        conference = ConferenceFactory.create()
+        conference = ConferenceFactory()
```

Comment:
As far as I can see, there isn't. I suspect that the __init__ function is simply calling .create() and doing some jiggery-pokery to make this work, but I haven't gone spelunking in the code. 
I admit that the *() form is a little weird, in that it looks like we're instantiating a ConferenceFactory object, which we aren't, but it feels a lot cleaner to me now that I've got used to it.